docker/Dockerfile.ubuntu.bionic

```diff
@@ -74,4 +75,7 @@ RUN make install
 
 RUN useradd testuser
 
+RUN ln -s /usr/bin/pip3 /usr/bin/pip
+RUN pip install pip-tools virtualenv wheel pytest
```

Comment:
This looks dangerous (the `ln` part). Can we just call `pip3 install`?

Reply:
Ah, sorry for totally misunderstood the goal. I will fix it.